test/e2e/density.go

```diff
@@ -181,15 +187,13 @@ var _ = Describe("Density", func() {
 		// TODO: Expose runLatencyTest as ginkgo flag.
 		{podsPerNode: 3, runLatencyTest: false, interval: 10 * time.Second},
 		{podsPerNode: 30, runLatencyTest: true, interval: 10 * time.Second},
-		// More than 30 pods per node is outside our v1.0 goals.
-		// We might want to enable those tests in the future.
 		{podsPerNode: 50, runLatencyTest: false, interval: 10 * time.Second},
-		{podsPerNode: 100, runLatencyTest: false, interval: 1 * time.Second},
+		{podsPerNode: 100, runLatencyTest: true, interval: 10 * time.Second},
 	}
 
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
-		if testArg.podsPerNode == 30 {
+		if testArg.podsPerNode == 100 {
```

Comment:
IIUC, we have a 1000-node cluster/kubemark that verifies the master components can handle 30000 pods already. In that case, for v1.2, I think we just want to test
 1. A node can handle 100 pods and can still meet the latency requirements
 2. A reasonably large cluster (e.g., 100 nodes) can handle 100 pods per node

For (1), I can simply add a new test to run in a regular suite.
For (2), I think we should change the scalability suite to run 100 pods per node. Starting a new suite doesn't help much because eventually we'll have to switch to 100 pods anyway. If you're worried about the possibility of breaking the suite, we can proceed by adding a new test that doesn't fail until it's stable. 

I'll change this PR to target (1). If you still prefer adding a new suite for (2), we can do that.